implementations/rust/ockam/ockam_command/src/tcp/connection/create.rs

```diff
@@ -77,13 +77,13 @@ pub async fn create_connection(
             let multiaddr = match route_to_multiaddr(&r) {
                 Some(addr) => addr,
                 None => {
-                    eprintln!("Couldn't convert given address into `MultiAddr`");
-                    std::process::exit(exitcode::SOFTWARE);
+                    let message = anyhow!("Couldn't convert given address into `MultiAddr`");
+                    return Err(Error::new(exitcode::SOFTWARE, message));
```

Comment:
Here too. In this case you can do: `route_to_multiaddr(&r).context("Couldn't convert given address into `MultiAddr`")?;